### PR TITLE
Create and Delete emoji commands

### PR DESCRIPTION
**Pull request by dpaiv0**

### CODE REVIEW

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -251,6 +251,21 @@
   "pause": {
     "commandDescription": "Pauses or resumes the music playback."
   },
+  "createemoji": {
+    "commandDescription": "Creates an emoji.",
+    "commandUsage": "<image url> <name>",
+    "noImage": "You need to give me an image URL address.",
+    "noName": "You need to give me a name for the emoji.",
+    "created": "Created emoji **{{emoji.name}}** successfully.",
+    "error": "An error has occurred while creating emoji"
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: Should it have punctuation?

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -251,6 +251,21 @@
...
+  },
+  "deleteemoji": {
+    "commandDescription": "Deletes an emoji.",
+    "commandUsage": "<:emoji:|emoji name>",
+    "noEmoji": "You need to give me the emoji you want to delete.",
+    "deleted": "Deleted emoji **{{emoji.name}}** successfully.",
+    "error": "An error has occurred while deleting emoji"
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: Should it have punctuation?

Comment (dpaiv0) [reply src/locales/en-US/commands.json]: I think so

File: src/locales/en-US/commands.json (hunk 1 above)
Comment (dpaiv0) [reply src/locales/en-US/commands.json]: yes

**Comment by dpaiv0** [comment]
This is pretty much

### RFR

**Pull request merged**